ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1940,6 +1940,35 @@ def test_tensorflow_rsqrt(
         x=x[0],
     )
 
+#segment_sum
+@handle_frontend_test(
+    fn_tree="tensorflow.math.segment_sum",
+    data=helpers.array_values(dtype=ivy.int32, shape=(5, 6), min_value=1, max_value=9),
+    segment_ids=helpers.array_values(
+        dtype=ivy.int32, shape=(5,), min_value=0, max_value=4
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_segment_sum(
+    *,
+    data,
+    segment_ids,
+    frontend,
+    test_flags,
+    fn_tree,
+    backend_fw,
+    on_device,
+):
+    helpers.test_frontend_function(
+        input_dtypes=["int32", "int64"],
```

Comment:
Here you should be passing the dtype that your data has. I recommend checking other tests to see how it is done